docs/General/Manuals/client-manual.rst

```diff
@@ -1362,10 +1362,43 @@ view will tell you what was sold where.
 Tax Rates View
 ^^^^^^^^^^^^^^
 
+The :guilabel:`Tax Rates View` is simply an icon on the `Top Function Bar`_ that shows you what percentage
+Gold, Science and Luxury Goods your nation is set at. Figure 38 is a sample of the icon for
+:guilabel:`Tax Rates View`.
```

Comment:
Consider rephrasing to "The Tax Rates View on the Top Function Bar shows what percentage of Gold, Science, and Luxury Goods your nation is set at. Figure 38 shows an example of the Tax Rates View."